azure/public_ip.sh

```diff
@@ -0,0 +1,24 @@
+# Public IP related functions
+
+fn azure_public_ip_create(name, group, location, allocation) {
+       (
+               az network public-ip create --name $name --resource-group $group --location $location --allocation-method $allocation
```

Comment:
I agree. This is actually not new, I just changed the azure-cli to 2.0. But since we are already changing the "package" name, I think we can add the error handling as well.